contracts/registry/src/lib.rs

```diff
@@ -123,35 +133,80 @@ impl Contract {
     /// Otherwise returns list of SBTs (identifed by issuer and list of token IDs) proving
     /// the `account` humanity.
     pub fn is_human(&self, account: AccountId) -> SBTs {
-        self._is_human(&account)
+        self._is_human(&account, false)
     }
 
-    fn _is_human(&self, account: &AccountId) -> SBTs {
-        if self._is_banned(&account) {
-            return vec![];
+    fn _is_human(&self, account: &AccountId, no_skip_ban: bool) -> SBTs {
+        if no_skip_ban {
+            if self.flagged.get(&account) == Some(AccountFlag::Blacklisted)
+                || self._is_banned(&account)
+            {
+                return vec![];
+            }
         }
-        if let Some(AccountFlag::Blacklisted) = self.flagged.get(&account) {
-            return vec![];
+
+        let proof = self._list_sbts_by_classes(account, &self.iah_sbts.0, &self.iah_sbts.1);
+        vec![(self.iah_sbts.0.clone(), proof)]
+    }
+
+    /// Returns tuple of community_verified proof and IAH proof.
+    /// If `is_human` is false, then the `is_human` check is skipped and empty list is
```

Comment:
this is not clear. I think we are returning is_community verified and empty is_human proof